source/frontend/src/components/maps/leaflet/Layers/util.tsx

```diff
@@ -183,9 +203,20 @@ export const getDraftIcon = (text: string) => {
  * @param feature the geojson object
  * @param latlng the point position
  */
-export const createSingleMarker = (feature: ICluster, latlng: LatLngExpression): Layer => {
-  const icon = getMarkerIcon(feature);
-  return new Marker(latlng, { icon });
+export const createSingleMarker = <P,>(
+  feature: Supercluster.PointFeature<P>,
+  latlng: LatLngExpression,
+): Layer => {
+  const isOwned = feature.id?.toString().startsWith('PIMS_PROPERTY_LOCATION_VW') ?? false;
```

Comment:
might be nice to have this as a function so if we come up with a better way of doing this we can fix all of these at once.